scripts/dnt/main.ts

```diff
@@ -0,0 +1,58 @@
+// ex. scripts/build_npm.ts
+import {
+  dirname,
+  fromFileUrl,
+  join,
+} from "https://deno.land/std@0.145.0/path/mod.ts";
+
+import {
+  build,
+  emptyDir,
+} from "https://raw.githubusercontent.com/denoland/dnt/main/mod.ts";
```

Comment:
Why not `https://deno.land/x/dnt/mod.ts`? Also, we should add a version number. This link would otherwise be a moving target and could randomly break the setup.